sanitizer/_text_addressit.js

```diff
@@ -1,10 +1,25 @@
-var check = require('check-types');
```

Comment:
yuss. I think now that we know and love lodash, we should eventually get rid of `check`. We are almost there